Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -434,13 +434,12 @@ def create(
         if component not in self.projects[project]['components']:
             raise ValueError("'{}' is not a recognized component in '{}'".format(component, project))
 
-        if not version and len(self.projects[project]['versions']) == 1:
-            version = self.projects[project]['versions'][0]
-        elif not version:
-            version = webkitcorepy.Terminal.choose(
-                "What version of '{}' should the bug be associated with?".format(project),
-                options=self.projects[project]['versions'], numbered=True,
-            )
+        if not version:
+            # This is the default option, aligned to webkit-patch behavior.
+            version = "WebKit Nightly Build"
```

Comment:
I think this should be taken as an argument to the Bugzilla class and then our repositories bug definition in `metadata/trackers.json`...everything in `Tools/Scripts/libraries/webkitbugspy/webkitbugspy` has managed to be project agnostic so far, seems like a shame to break that now (even though bugzilla, in particular, is really only relevant to WebKit...)

Reply:
OK, for now, I'll land it as is to fix the regression requiring the version in git-webkit, and adding a FIXME here.